hoomd/md/pytest/test_custom_force.py

```diff
@@ -378,3 +378,39 @@ def test_torques_update(local_force_names, two_particle_snapshot_factory,
         if sim.device.communicator.rank == 0:
             assert np.count_nonzero(snap.particles.orientation
                                     - initial_orientations)
+
+
+def test_force_zeroing(force_simulation_factory, two_particle_snapshot_factory):
+
+    class TestForceZeroing(hoomd.md.force.Custom):
+
+        def __init__(self):
+            super().__init__(aniso=True)
+
+        def set_forces(self, timestep):
+            with self.cpu_local_force_arrays as force:
+                force.force[:] += timestep
+                force.torque[:] += timestep
+                force.virial[:] += timestep
+
+    snap = two_particle_snapshot_factory()
+    test_force = TestForceZeroing()
+    sim = force_simulation_factory(test_force, snap)
+    if sim.device.communicator.rank == 0:
+        snap.particles.moment_inertia[:] = [[1, 1, 1], [1, 1, 1]]
+    sim.state.set_snapshot(snap)
+
+    _skip_if_gpu_device_and_no_cupy(sim)
+    sim.operations.integrator.integrate_rotational_dof = True
+    sim.always_compute_pressure = True
+
+    for _ in range(3):
+        sim.run(1)
+        forces = test_force.forces
+        torques = test_force.torques
+        virials = test_force.virials
```

Comment:
Just from the blob of the class TestForceZeroing, should there be `s` on test_force.forces/torques/virials?

Reply:
Yes, the Python and C++ APIs use slightly different names. In Python, it is customary to use the plural form to denote lists or arrays as a Python variable may be of any type. This is not necessary in C++ where every variable has a well defined type and that type makes it clear whether the variable represents an array or a single item.

Thanks for taking the time to review and to note a potential problem.